app/src/main/java/com/sun/weather/screen/setting/SettingFragment.kt

```diff
@@ -14,21 +16,29 @@ class SettingFragment : BaseFragment<FragmentSettingBinding>() {
     }
 
     override fun initData() {
-        // TODO LATER
-    }
-
-    override fun initView() {
         val flagResId = SharedPrefManager.getInt(KEY_FLAG_RES_ID, DEFAULT_FLAG_RES_ID)
         viewBinding.flagImageView.setImageResource(flagResId)
         viewBinding.languageButton.setOnClickListener {
             showLanguageSelectionDialog()
         }
     }
 
+    override fun initView() {
+        viewBinding.flagImageView.setImageResource(R.drawable.vn_flag)
+    }
+
     private fun showLanguageSelectionDialog() {
+        val languageCode = Locale.getDefault().language
+        val languages =
+            when (languageCode) {
+                LANGUAGE_CODE_VIETNAMESE -> arrayOf("Tiếng Việt", "Tiếng Anh")
+                LANGUAGE_CODE_ENGLISH -> arrayOf("VietNamese", "English")
+                else -> arrayOf("VietNamese", "English")
```

Comment:
```suggestion
                LANGUAGE_CODE_VIETNAMESE -> arrayOf("Tiếng Việt", "Tiếng Anh")
                LANGUAGE_CODE_ENGLISH -> arrayOf("VietNamese", "English")
                else -> arrayOf("VietNamese", "English")
```
dùng [String Array](https://developer.android.com/guide/topics/resources/string-resource#StringArray) để localize